tests/unit/disco/components/TestFooter.js

```diff
@@ -0,0 +1,26 @@
+import { shallow } from 'enzyme';
+import React from 'react';
+
+import { FooterBase } from 'disco/components/Footer';
+import { getFakeI18nInst } from 'tests/unit/helpers';
+
+
+function shallowRender(props) {
```

Comment:
Yeah it handles it and I feel it's nicer to write it like this... I've been doing it in components lately. The boilerplates of empty objects is annoying.